newdemojail.yaml

```
@@ -0,0 +1,772 @@
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "OfficerPhoneNo"
+      nc.PersonGivenName:
+        type: "nc.PersonGivenName"
+        column: "OfficerPhoneNo"
+      nc.PersonMiddleName:
+        type: "nc.PersonMiddleName"
+        column: "OfficerPhoneNo"
+      nc.PersonSurName:
+        type: "nc.PersonSurName"
+        column: "OfficerPhoneNo"
+    conditions:
+    - !<conditions.BooleanRegexCondition>
+      column: "HearingType"
+      pattern: "Probation"
+      reverse: false
+    name: "probationofficer"
+
+  paroleattorney:
+    fqn: "general.person"
+    entitySetName: "DemoParoleAttorneys"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "AttorneyPhoneNo"
+      nc.PersonGivenName:
+        type: "nc.PersonGivenName"
+        column: "AttorneyFirstName"
+      nc.PersonMiddleName:
+        type: "nc.PersonMiddleName"
+        column: "AttorneyMiddleName"
+      nc.PersonSurName:
+        type: "nc.PersonSurName"
+        column: "AttorneyLastName"
+    conditions:
+    - !<conditions.BooleanRegexCondition>
+      column: "HearingType"
+      pattern: "Parole"
+      reverse: false
+    name: "paroleattorney"
+
+  probationattorney:
+    fqn: "general.person"
+    entitySetName: "DemoProbationAttorneys"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "AttorneyPhoneNo"
+      nc.PersonGivenName:
+        type: "nc.PersonGivenName"
+        column: "AttorneyFirstName"
+      nc.PersonMiddleName:
+        type: "nc.PersonMiddleName"
+        column: "AttorneyMiddleName"
+      nc.PersonSurName:
+        type: "nc.PersonSurName"
+        column: "AttorneyLastName"
+    conditions:
+    - !<conditions.BooleanRegexCondition>
+      column: "HearingType"
+      pattern: "Probation"
+      reverse: false
+    name: "probationattorney"
+
+  attorney:
+    fqn: "ol.occupation"
+    entitySetName: "DemoAttorneys"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "AttorneyPhoneNo"
+      ol.type:
+        type: "ol.type"
+        column: "HearingType"
+    name: "attorney"
+  
+  officer:
+    fqn: "ol.employee"
+    entitySetName: "DemoOfficers"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "OfficerPhoneNo"
+      ol.category:
+        type: "ol.category"
+        column: "HearingType"
+    name: "officer"
+
+  officercontactinformation:
+    fqn: "ol.contactinformation"
+    entitySetName: "DemoOfficerContacts"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "OfficerPhoneNo"
+      contact.phonenumber:
+        type: "contact.phonenumber"
+        column: "OfficerPhoneNo"
+      staff.email:
+        type: "staff.email"
+        column: "OfficerEmail"
+      ol.type:
+        type: "ol.type"
+        column: "HearingType"
```

```suggestion
   #   ol.type:
   #    type: "ol.type"
   #    column: "HearingType"
```
This is for a different kind of purpose, like the type of contact info being recorded, if we need more elaboration there.

---

Removed